examples/oneapi/cpp/source/example_util/output_helpers_graph.hpp

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include <iomanip>
+#include <iostream>
+#include <numeric>
+
+#include "oneapi/dal/table/common.hpp"
+#include "oneapi/dal/algo/jaccard/vertex_similarity_types.hpp"
+#include "oneapi/dal/table/homogen.hpp"
+
+void print_vertex_similarity_result(
```

Comment:
We assume that helpers can be reused between several components. 
Can we make it less specific? Now it's targeted only for similarity and only for jaccard algorithm. What if we want to print out the same similarity results for other graph algorithms?